app/test/helpers/random-data.ts

```diff
@@ -0,0 +1,12 @@
+import { randomBytes } from 'crypto'
+
+/**
+ * Generate a pseudo-random sequence of hexadecimal values for use in testing.
+ *
+ * Source: https://blog.abelotech.com/posts/generate-random-values-nodejs-javascript/
+ */
+export function generateString(length: number = 32) {
```

Comment:
does this overlap with #7499? it looks like b19af067816ce760bf643f8ccf8bba1d217c8215 is in both PRs

Reply:
Yep, and not needed here. Let me update the PR to drop this work.